### PR TITLE
aws: bedrock processor

### PR DESCRIPTION
**Pull request by rockwotj**
Add an AWS Bedrock Processor similar to the other AI processors that
exist out there.

### CODE REVIEW

File: internal/impl/aws/processor_bedrock.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+		Field(service.NewStringField(bedFieldModel).
+			Examples("amazon.titan-text-express-v1", "anthropic.claude-3-5-sonnet-20240620-v1:0", "cohere.command-text-v14", "meta.llama3-1-70b-instruct-v1:0", "mistral.mistral-large-2402-v1:0").
+			Description("The model ID to use. For a full list see the https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html[AWS Bedrock documentation^].")).
+		Field(service.NewStringField(bedFieldUserPrompt).
+			Description("The prompt you want to generate a response for. By default, the processor submits the entire payload as a string.").
+			Optional()).
+		Field(service.NewStringField(bedFieldSystemPrompt).
+			Optional().
+			Description("The system prompt to submit to the AWS Bedrock LLM.")).
+		Field(service.NewIntField(bedFieldMaxTokens).
```
Comment (mihaitodor) [inline internal/impl/aws/processor_bedrock.go:57]: I think this has to be at least 1: https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_InferenceConfiguration.html#bedrock-Type-runtime_InferenceConfiguration-maxTokens

**Review by mihaitodor — APPROVED** [review]
Nice work Tyler! Just left a few small nits. Feel free to :shipit:

File: internal/impl/aws/processor_bedrock.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+const (
+	bedFieldModel        = "model"
+	bedFieldUserPrompt   = "prompt"
+	bedFieldSystemPrompt = "system_prompt"
+	bedFieldMaxTokens    = "max_tokens"
+	bedFieldTemp         = "stop"
+	bedFieldStop         = "temperature"
+	bedFieldTopP         = "top_p"
+)
```
[the remark this answers is not in the document]
Comment (rockwotj) [reply internal/impl/aws/processor_bedrock.go]: Haha thanks for keeping me consistent :smile:

File: internal/impl/aws/processor_bedrock.go (hunk 1 above)
Comment (rockwotj) [reply internal/impl/aws/processor_bedrock.go]: Oh nice catch - I didn't even think someone would try that. Added a lint rule

**Pull request merged**